src/main/java/uk/gov/pay/directdebit/gatewayaccounts/resources/GatewayAccountResource.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.pay.directdebit.gatewayaccounts.resources;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+import uk.gov.pay.directdebit.gatewayaccounts.api.CreateGatewayAccountValidator;
+import uk.gov.pay.directdebit.gatewayaccounts.api.GatewayAccountResponse;
+import uk.gov.pay.directdebit.gatewayaccounts.model.GatewayAccount;
+import uk.gov.pay.directdebit.gatewayaccounts.services.GatewayAccountService;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+@Path("/")
```

Comment:
Do we need this path at root level? If not, maybe it's good to remove them from all resources (probably in a separate clean up commit). Thanks